Source/Core/InputCommon/ControllerEmu.h

```diff
@@ -406,12 +407,36 @@ class ControllerEmu
           yy += (numeric_settings[0]->GetValue() - 0.5);
         }
 
-        *x = xx;
-        *y = yy;
+        // relative input
+        if (boolean_settings[0]->GetValue())
+        {
+          const ControlState deadzone = numeric_settings[3]->GetValue();
+          // deadzone to avoid the cursor slowly drifting
+          if (std::abs(xx) > deadzone)
+            m_x = MathUtil::Clamp(m_x + xx * SPEED_MULTIPLIER, -1.0, 1.0);
+          if (std::abs(yy) > deadzone)
+            m_y = MathUtil::Clamp(m_y + yy * SPEED_MULTIPLIER, -1.0, 1.0);
+        }
+        else
+        {
+          m_x = xx;
+          m_y = yy;
+        }
+
+        *x = m_x;
+        *y = m_y;
       }
     }
 
     ControlState m_z;
+
+  private:
+    // This is used to reduce the cursor speed for relative input
+    // to something that makes sense with the default range.
+    static constexpr double SPEED_MULTIPLIER = 0.02;
```

Comment:
Did some more testings and I think we should increase this value a little. With the default range (100) the cursor movement is a bit on the slow side and at the maximum range (500) it's still a little too slow. I haven't compiled this PR locally to test but I think 0.04 is probably a sweet spot, at least theoretically...